data/mods/Magiclysm/Spells/magus.json

```diff
@@ -408,5 +408,31 @@
     "base_energy_cost": 350,
     "energy_source": "MANA",
     "difficulty": 4
+  },
+  {
+    "id": "focused_bolt",
+    "type": "SPELL",
+    "name": "Focused Bolt",
+    "description": "You focus your mana into a microscopically thin pulsed beam of pure energy, capable of extreme range and precision;  as well as unparalleled penetration, provided proper skill.  Once the invention of a notorious assassin, this spell has slain many a king and noble.  Passed on only through direct descendants over the ages, though occasionally shared by them to others.",
+    "effect": "attack",
+    "shape": "line",
+    "valid_targets": [ "hostile", "ground" ],
+    "flags": [ "SOMATIC", "NO_LEGS", "CONCENTRATE", "SILENT", "NO_PROJECTILE" ],
+    "max_level": 30,
+    "min_damage": 4,
+    "max_damage": 124,
+    "damage_increment": 4.0,
+    "damage_type": "pure",
+    "min_range": 5,
+    "max_range": 75,
```

Comment:
i can't recall the max range you can see on the map view, but isn't it more like 50 or 60 tiles?

Reply:
yeah 50 range, ill reduce it